client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -239,8 +238,10 @@ export class HubConnection {
         }
     }
 
-    set onClosed(callback: ConnectionClosed) {
-        this.connectionClosedCallback = callback;
+    onClosed(callback: ConnectionClosed) {
```

Comment:
I prefer onclose

Reply:
This is for consistency - we use camelCase for all APIs so it would be weird to use lowercase here. (Granted, most of methods have just one words so it is not as visible)

Reply:
> (Granted, most of methods have just one words so it is not as visible)

Which makes that point moot 😄 . onclose looks more like other browser APIs as well.

https://www.tutorialspoint.com/html5/html5_websocket.htm

Looks more javascripty.

Reply:
We already had this discussion and we decided to use camelCase because the old JS client [uses camelCase](https://github.com/SignalR/SignalR/blob/d463e0967edb416362c27b58e0a9515cbae698fe/src/Microsoft.AspNet.SignalR.Client.JS/jquery.signalR.core.js#L54-L63). What I am trying to say is that if we now want event names to be lowercase we should probably change it everywhere (i.e. `IConnection` and `ITransport`) and not only here.

(I find it quite amusing you bring webocket API to make your point in a PR where we change the onClose property to a function since "it is a better API" while websocket do use properties and not functions for
 events 🤣)

Reply:
😄 Camel case looks so weird for these methods. I dunno what it is but it feels like it doesn't fit.

Reply:
I am fine with changing it everywhere.